test/gtest/uct/uct_p2p_test.cc

```diff
@@ -173,7 +173,7 @@ void uct_p2p_test::test_xfer_multi_mem_type(send_func_t send, size_t min_length,
     ms << "memory_type:" << uct_test::uct_mem_type_names[mem_type] << " " << std::flush;
 
     /* Trim at 4GB */
-    max_length = ucs_min(max_length, 4ull * 1124 * 1024 * 1024);
+    max_length = ucs_min(max_length, 4ull * UCS_GBYTE);
```

Comment:
it's supposed to be larger than 4 GB

Reply:
`/* Trim at 4GB */` comment says that the code below trims `max_lentgh` at `4 Gb`.

Reply:
sorry, this comment is misleading. it should be "trim at 4.1 GB"

Reply:
> sorry, this comment is misleading. it should be "trim at 4.1 GB"

now trim at `4.1 * UCS_GBYTE` with an appropriate cast
also the comment has been fixed